fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -503,6 +505,21 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun setScrollingBehavior() {
```

Comment:
Sorry for the being pedantic but the new approach that will disable dragging in all situations I think warrants being specific about the intent.
Can you please rename to something like `disableAppBarDragging()`. 
It's an important behavior change that I think we need to highlight.